packages/css-framework/src/components/_listitem.scss

```diff
@@ -2,7 +2,7 @@
 //   // border-bottom: 1px solid color(40);
```

Comment:
Redundant styles?

Reply:
No idea - the whole of `_listitem.scss` seems to be commented out. Presuming it's safe to drop?

Reply:
Yeah - I think these are maybe styles leftover from the legacy library.